ex03.ex

```diff
@@ -104,8 +118,14 @@ defmodule Ex03 do
       false
 
   """
-
-  def list_equal . . . "your code"
+  def list_equal([], []), do: true
```

Comment:
# -1

Same comment here. It could just be:

~~~ elixir
  def list_equal([], []),               do: true
  def list_equal([ h | t1 ], [h | t2]), do: list_equal(t1, t2)
  def list_equal(_, _),                 do: false
~~~